### PR TITLE
OCLOMRS-228:Change the UI on the pages to start with create then cancel

### PR DESCRIPTION
**Pull request by Michyjones**
# JIRA TICKET NAME:
[OCLOMRS-228: Change the UI on the pages to start with create then cancel](https://issues.openmrs.org/browse/OCLOMRS-228)

# Summary:
The cancel button came before the create button. This is not a good practice. Therefore, this ticket was to ensure the create or add button comes before cancel or close button

### CODE REVIEW

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 1]
```diff
@@ -44,55 +44,56 @@ const CreateConceptForm = props => (
               onChange={props.handleChange}
             >
               <option value="N/A" />
-              {classes.map(option => <option key={option}>{option}</option>)}
+              {classes.map(option => (
+                <option key={option}>{option}</option>
```
Comment (kmunene) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: What happened here?

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 2]
```diff
@@ -44,55 +44,56 @@ const CreateConceptForm = props => (
...
+                    className="form-control symptom-finding"
+                    required
+                    onChange={props.handleChange}
+                  >
+                    <option key="Symptom-Finding">Symptom-Finding</option>
+                    <option key="Symptom">Symptom</option>
+                    <option key="Finding">Finding</option>
+                  </select>
+                </React.Fragment>
+              )}
```
[the remark this answers is not in the document]
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: @alexmochu  When you are creating symptom-finding concepts, under class there are those selections. You can choose symptom-finding, symptom and also finding. So this part is for selecting Finding only.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 1 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: @kmunene This is eslint fix.

Comment (dondrzzy) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Can you please revert this formatting change as it looks like it is not part of this ticket.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (dondrzzy) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: @Michyjones, Is this part of this ticket? If so, how does it affect the switching between the create and cancel buttons?

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 3]
```diff
@@ -44,55 +44,56 @@ const CreateConceptForm = props => (
...
 
-              {(props.concept.toString().trim() !== 'symptom-finding' && props.concept.toString().trim() !== 'set') && (
-              <span className="btn btn-sm btn-light normal-cursor col-12 text-capitalize pt-3">
-                {props.concept}
-              </span>
+              {props.concept.toString().trim() === 'set' && (
```
Comment (dondrzzy) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Same goes for this.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 4]
```diff
@@ -125,7 +126,12 @@ const CreateConceptForm = props => (
         <div className="row col-12 custom-concept-list">
           <h6 className="text-left section-header">Names</h6>
           <CreateConceptTable {...props} />
-          <a href="#!" className="text-left add-more-names" id="add-more-name" onClick={props.handleNewName}>
+          <a
```
Comment (dondrzzy) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: This looks like a formatting change. Can this be done in a different PR?

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (kmunene) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: I agree with @dondrzzy on this.

**Comment by dkayiwa** [comment]
@Michyjones why has this taken more than a week without closing?

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: This is just fixing elslint warning.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 3 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Same here fixing elslint warning.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 4 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: This is just fixing elslint warning.

**Comment by dkayiwa** [comment]
@Michyjones why is this still open?

**Comment by dkayiwa** [comment]
@Michyjones did you inform me that you are still waiting for your team mates to review, as we had agreed?

**Comment by Michyjones** [comment]
@dkayiwa  No, apologies for that. I will constantly be keeping on the loop. Regular updates will be coming from today moving forward.

**Comment by Michyjones** [comment]
@dkayiwa  It took more than a week because it was no reviewed on time and also I had not implemented all the feedback given and  resolving merge conflicts

**Comment by dkayiwa** [comment]
@Michyjones did you tell me when it was not reviewed on time?

**Comment by Michyjones** [comment]
@dkayiwa  I did not communicate. I will improve on my communication and constantly update you.

**Comment by dkayiwa** [comment]
@Michyjones so where are we?

**Comment by Michyjones** [comment]
@dkayiwa I have reached out to the teammate and there were some formatting changes I had not handle.

**Comment by dkayiwa** [comment]
@Michyjones so have they now reviewed it within our accepted time?

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexmochu) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: @Michyjones Then create a ticket to handle that eslint fix. Was this part of your scope?

**Comment by Michyjones** [comment]
@dkayiwa  Yes, they reviewed it but not as we had agreed on 1 day or at most 2 days

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 3 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: I reverted all the changes that are not part of this ticket.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 4 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: I reverted all the changes that are not part of this ticket.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 1 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: @dondrzzy I have reverted the formatting changes

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 3 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: @dondrzzy  Concerning the above question. That is not part of the ticket that is why I have reverted the changes.

**Comment by Michyjones** [comment]
@dkayiwa  This PR took more than a week as I was waiting for the reviews and also implementing the feedback given by the reviewers.

**Comment by dkayiwa** [comment]
@Michyjones are all these changes about simply switching positions between create and cancel?

**Comment by Michyjones** [comment]
@dkayiwa Yes, I had to change some files.

**Comment by dkayiwa** [comment]
@Michyjones all of these? Are you sure?

**Comment by Michyjones** [comment]
There two set of button I was  changing, The other part is on the preview concept where  add and close button

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 5]
```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
...
             <label htmlFor="class">Class</label>
             <div>
-              { (props.concept.toString().trim() === 'symptom-finding') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control symptom-finding"
-                  required
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Can you explain how this change is related to the ticket?

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 6]
```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
...
-                  <option key="Symptom">Symptom</option>
-                  <option key="Finding">Finding</option>
-
-                </select>
-              </React.Fragment>
+              {(props.concept.toString().trim() === 'symptom-finding') &&
+                <React.Fragment>
+                  <select
+                    id="class"
+                    name="concept_class"
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Same here

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 7]
```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
...
+                </React.Fragment>
               }
-
-              { (props.concept.toString().trim() === 'set') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control set"
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Same here

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 8]
```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
...
-                  <option key="MedSet">MedSet</option>
-
-                </select>
-              </React.Fragment>
+              {(props.concept.toString().trim() === 'set') &&
+                <React.Fragment>
+                  <select
+                    id="class"
+                    name="concept_class"
+                    value={props.state.concept_class}
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Same here

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx [hunk 9]
```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
...
+                    <option key="ConvSet">ConvSet</option>
+                    <option key="MedSet">MedSet</option>
+                  </select>
+                </React.Fragment>
               }
 
-              {(props.concept.toString().trim() !== 'symptom-finding' && props.concept.toString().trim() !== 'set') && (
-              <span className="btn btn-sm btn-light normal-cursor col-12 text-capitalize pt-3">
+              {props.concept.toString().trim() !== 'symptom-finding' &&
+props.concept.toString().trim() !== 'set' && (
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Same here

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 5 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: All this are formatting changes that I was reverting. They are not part of this ticket.

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 6 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Same here the changes that I reverted

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 7 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: All this are formatting changes that I was reverting. They are not part of this ticket.

Comment (dkayiwa) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: But i see them in this commit. This pull request should not have any change that is not related to what the ticket requires. If you need formatting, use a new ticket and pull request.

Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: I can just create a ticket for just format fixing?

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 8 above)
Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Let me revert all the formatting  changes and I will create a separate ticket  for it

File: src/components/dictionaryConcepts/components/CreateConceptForm.jsx (hunk 7 above)
Comment (dkayiwa) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: yes you can

Comment (Michyjones) [reply src/components/dictionaryConcepts/components/CreateConceptForm.jsx]: Let me make the changes I will get back to you in a few.

**Comment by Michyjones** [comment]
@dkayiwa I have made the changes

**Comment by Michyjones** [comment]
I will create a ticket ti fix the formatting issues because eslint is complaining

**Comment by dkayiwa** [comment]
@Michyjones can you create the ticket and put a link to it here?

**Comment by Michyjones** [comment]
@dkayiwa I have made the changes now it looks like what is on the master. I was comparing as I was making the changes.

**Comment by dkayiwa** [comment]
I have not see any change in the commit on this pull request. It still has lots of formatting changes. Can you ask other apprentices for help?

**Comment by Michyjones** [comment]
@dkayiwa  sorry I took long I was on transit. The ticket now contain the changes that are required.
